### ISSUE TITLE
Unknown trace events with clang 11

### ISSUE THREAD

**Issue by sigiesec** [aras-p/ClangBuildAnalyzer#56]
When using ClangBuildAnalyzer, there are lots of "unknown trace event" warnings, including:
```
PassManager<llvm::Function>
PassManager<llvm::Module>
LoopLoadEliminationPass
Float2IntPass
ModuleToFunctionPassAdaptor<llvm::PassManager<llvm::Function>>
GlobalDCEPass
CGProfilePass
ConstantMergePass
```

Not sure if these are meaningful to handle in ClangBuildAnalyzer, but at least they should not produce warnings.

**Comment by aras-p** [comment]
Should be fixed by #80 (merged into main branch, I'll tag the 1.2.0 release binaries soon)
